x-pack/plugins/infra/server/lib/alerting/metric_threshold/metric_threshold_executor.ts

```diff
@@ -177,15 +179,6 @@ export const createMetricThresholdExecutor = (libs: InfraBackendLibs) =>
             })
           )
           .join('\n');
-        /*
-         * Custom recovery actions aren't yet available in the alerting framework
-         * Uncomment the code below once they've been implemented
```

Comment:
@simianhacker I see you removed the commented code for recovery actions. My understanding is that since this PR is merged https://github.com/elastic/kibana/pull/124972 (Provide services to set context for recovered alerts), we can now build recovered alert reason, or not yet? Do we have a ticket to track what you just removed?

Reply:
https://github.com/elastic/kibana/issues/127078